libs/repositories/src/repos/UsdRepository/UsdRepositoryCow.ts

```diff
@@ -71,6 +94,22 @@ export class UsdRepositoryCow extends UsdRepositoryNoop {
     return usdcPrice.div(tokenPrice).toNumber();
   }
 
+  private getChainId(chainIdOrSlug: string) {
```

Comment:
this is a static method, should not be a class method. Maybe move to an util?

But I think is also too complicated, not? 
Why not just `ALL_CHAINS_IDS.includes(Number(chainIdOrSlug))`

Reply:
Fixed, extracted to utls and simplified